keyboards/1upkeyboards/1up60hse/keymaps/via/keymap.c

```diff
@@ -0,0 +1,82 @@
+   * |-----------------------------------------------------------------------------------------+
+   * |         |Left |Down |Right|     |     |     |     |     | Ins |Home |PgUp |             |
+   * |-----------------------------------------------------------------------------------------+
+   * |           |VolUp|VolDn|VolMu|     |     |     |     |     | End |PgDn |                 |
+   * |-----------------------------------------------------------------------------------------+
+   * |       |       |       |                                 |       |  L2   |       |       |
+   * `-----------------------------------------------------------------------------------------'
+   */
+  [1] = LAYOUT_60_ansi(
+    KC_GRV, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_DEL,
+    KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_TRNS,
+    KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_INS, KC_HOME, KC_PGUP, KC_TRNS,
+    KC_TRNS, KC_VOLU, KC_VOLD, KC_MUTE, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_END, KC_PGDN, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, MO(2), KC_TRNS, KC_TRNS
+  ),
+
+  /* RGB
+   * ,-----------------------------------------------------------------------------------------.
+   * |     |     |     |     |     |     |     |     |     |     |     |     |     |   Reset   |
+   * |-----------------------------------------------------------------------------------------+
+   * | BL Tog |BLInc|BLDec|BLStp|     |     |     |     |     |     |     |     |     |        |
+   * |-----------------------------------------------------------------------------------------+
+   * | RGB Tog |Mode |Hue I|Sat I|Val I|Spd I|Plain|Breat|Rnbow|Swirl|     |     |             |
+   * |-----------------------------------------------------------------------------------------+
+   * |           |RMode|Hue D|Sat D|Val D|Spd D|Snake|Knigh|Xmas |Gradi|     |                 |
+   * |-----------------------------------------------------------------------------------------+
+   * |       |       |       |                                 |       |       |       |       |
+   * `-----------------------------------------------------------------------------------------'
+   */
+  [2] = LAYOUT_60_ansi(
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RESET,
+    BL_TOGG, BL_INC, BL_DEC, BL_STEP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    RGB_TOG, RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, RGB_M_P, RGB_M_B, RGB_M_R, RGB_M_SW, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS, RGB_RMOD, RGB_HUD, RGB_SAD, RGB_VAD, RGB_SPD, RGB_M_SN, RGB_M_K, RGB_M_X, RGB_M_G, KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+  ),
```

Comment:
VIA uses 4 layers by default, add a 4th layer of `KC_TRNS`.

Reply:
Interesting, I didn't know we had to define it even if unused for VIA.

Reply:
They're used as the default keycodes for dynamic keymaps. I can't determine how many layers are in that keymap in flash, so it needs to match the number of layers used for dynamic keymaps.